rpc-client/src/nonblocking/rpc_client.rs

```diff
@@ -2197,12 +2199,20 @@ impl RpcClient {
             current_percent = 100f32 * max as f32 / total_active_stake as f32;
             if current_percent < max_stake_percent {
                 break;
```

Comment:
This interruption condition is confusing to me.
At first, I thought the function allows one to wait for a certain number of stake to be active.

This code goes through all the vote accounts, both active and delinquent, and picks an account with the most amount of stake.
Then it computes the percentage of this stake relative to all active stake.

But we exit if this percentage is **below** the requested expectation.
Shouldn't it be the opposite?

Reply:
Brennan explained that the behavior is actually as intended.  It is just that the function name is confusing.